### PR TITLE
Fix: Linting errors with upper-case variables

### PR DESCRIPTION
**Pull request by emilarista**
## Change Summary

Fixes patch panel doc template to not use capital letters in j2 variables.

## Component(s) name

`arista.avd.eos_cli_config_gen`

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]
LGTM. We will hold this PR back until we have merged the J2lint. Then we need to enable the j2lint for lower case variables as part of this PR.

**Review by carlbuchmann — APPROVED** [review]
LGTM!

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**
